static/docs/commands-reference/checkout.md

```diff
@@ -122,7 +124,7 @@ $ cd example-get-started
 
 </details>
 
-The workspace looks almost like in this
+The workspace `tree` looks almost like in this
```

Comment:
I'm not sure we need to mention this explicitly tbh

Reply:
Removed in e0e0285.